packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -30,53 +38,214 @@ export const useTooltip = (
   ref: React.Ref<HTMLElement>,
   defaultProps?: TooltipProps,
 ): TooltipState => {
-  const [target, setTarget] = React.useState<HTMLElement | null>(null);
+  const [visible, setVisible] = React.useState(false);
```

Comment:
You might want to use `useBoolean` from `@fluentui/react-hooks`.

Reply:
It doesn't look like that hook adds tangible benefit in this case, and it adds overhead beyond a simple `useState` call.